<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlarmClock</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/trayicon.png</normaloff>:/trayicon.png</iconset>
  </property>
  <widget class="QLineEdit" name="portLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hostComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="serverNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="serverPortLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>79</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fill the fields below to connect to the server:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>120</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>120</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
